components/config/opts.rs

```diff
@@ -143,6 +143,9 @@ pub struct Opts {
     /// Whether we're running inside the sandbox.
     pub sandbox: bool,
 
+    /// Whether to disable the http cache.
+    pub disable_http_cache: bool,
```

Comment:
Rather than a command-line flag, we should use runtime preferences that are controlled by [prefs.json](https://github.com/servo/servo/blob/master/resources/prefs.json) and accessed by the [preferences api](https://github.com/servo/servo/blob/4cf2ce66fc4f970a47ab1fb4b9aa1a55282640f7/components/script/dom/mouseevent.rs#L176). This also allows us to write a WPT test that turns off the cache if we want: https://github.com/servo/servo/blob/234951e4b6d66891cd0cc9705f42e3ca16d9e3d5/tests/wpt/mozilla/meta/mozilla/mozbrowser/__dir__.ini#L1 . If we do that, it should live in tests/wpt/mozilla/tests/mozilla.